kube-core/src/response.rs

```diff
@@ -34,6 +38,11 @@ pub struct Status {
 }
 
 impl Status {
+    /// Returns a boxed `Status`
+    pub fn boxed(self) -> Box<Self> {
+        Box::new(self)
+    }
```

Comment:
I guess this needs to be public because it's needed to raise the error across crates.

Reply:
Right.